UKHomeOffice/nmsw-ui
# Test other amg responses

Test-NMSW-1375 UPT content update
Test NMSW-1356 verify error message if user tries to submit AMG errors exist
Test NMSW-1373 -Change UPT status to display ResponseSector wording as tag
Test NMSW-1374 verify ETA sign in cookie refactor

## To test
Run with cypress runner
```
npm run cypress:open:dev -- --env MAIL_API_KEY=${MAIL_API_KEY}
```
and choose amg-error-validation.feature
passenger-status-check.feature
task-details-page.feature
or in cmd line
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/passenger-status-check.feature
```
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/amg-error-validation.feature
```
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/task-details-page.feature
```
